### DISCUSSION TITLE
Place editor widget on first place

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10691] [Q&A]
Hi,
I wish to know if there is any to possible to set a `CustomEditorWidget` to be placed first and not at last upon opening based on condition using `WidgetOptions` in `CustomEditorManager`. 

Thanks

**Comment by colin-grant-work** [comment]
Hi, @maheshrajrp, the code that handles the attachment of custom editor widgets is here:

https://github.com/eclipse-theia/theia/blob/0ad6d10e354af57b43fa7ac677c43c8a4948e494/packages/plugin-ext/src/main/browser/custom-editors/custom-editors-main.ts#L217-L275

It looks like it currently doesn't support placing an editor first in its group - it is either last or `beside` some other widget reference - but it could be made more flexible.
